contrastsecurity/README.md

```diff
@@ -0,0 +1,44 @@
+## Contrast Security - Protect ScreenBoard for DataDog
+
+### Set up Contrast Protect logs collection
+
+Enable logs collection for Datadog Agent in `/etc/datadog-agent/datadog.yaml` on Linux platforms. On other platforms, refer to the [Agent Configuration Files guide](https://docs.datadoghq.com/agent/guide/agent-configuration-files/?tab=agentv6) for the location of your configuration file:
+```
+logs_enabled: true
+```
+
+* Create a new folder `java.d` in the `conf.d/` directory of DataDog configuration directory.
+* Create a new `conf.yaml` file.
+* Add a custom log collection configuration group.
+```
+logs:
+  - type: file
+    path: /path/to/contrast/security.log
+    service: contrast
+    source: java
+```
+
+* [Restart the Datadog Agent](https://docs.datadoghq.com/agent/guide/agent-commands/?tab=agentv6#restart-the-agent).
+
+For more information, consult the [Logs Collection documentation](https://docs.datadoghq.com/logs/log_collection/?tab=tailexistingfiles#getting-started-with-the-agent).
+
+### Create a Logs Processing Pipeline
+
+* Create a new pipeline in Datadog (*Logs* -> *Configuration* - > *New Pipeline*) 
+* Expand the new pipeline and click *Add Processor*.
+* Add a Grok parser to the pipeline with the following parsing rule:
+```
+ContrastSecurityLogRule %{data:data}pri=%{data:pri} src=%{ip:src} spt=%{number:spt} request=%{data:request} requestMethod=%{word:requestMethod} app=%{data:app} outcome=%{word:outcome}
+```
+
+### Import Contrast Security dashboard into Datadog
+You can find your API key and application keys in *Integrations* -> *APIs*.
+
+```bash
+api_key=<YOUR_API_KEY>
+app_key=<YOUR_APPLICATION_KEY>
+
+curl  -X POST -H "Content-type: application/json" -d @contrast_security_protect.json "https://api.datadoghq.com/api/v1/dashboard?api_key=${api_key}&application_key=${app_key}"
```

Comment:
Where is `contrast_security_protect.json`? If you include it in the integration and declare it it on the manifest we can create an in-app tile that will allow users to install the screenboard with the click of a button. Note that to do that the json file needs to follow the same format as the ones you find in `integrations-core` repo.

*IMPORTANT*: The dashboard payload that gets placed into the integration folder MUST come from the /screen API endpoint and not the /dash endpoint.

Reply:
i added it!

Reply:
Now we do not installing dashboard step :)

Reply:
Deleted! And short description is there!